src/main/java/org/spongepowered/api/GameRegistry.java

```diff
@@ -85,6 +85,23 @@
 public interface GameRegistry {
 
     /**
+     * Creates a catalog key.
+     *
+     * @param namespace The namespace
+     * @param value The value
+     * @return A new catalog key
+     */
+    CatalogKey createKey(final String namespace, final String value);
```

Comment:
Ignore what I said...need somewhere to point the static of methods in CatalogKey to. I guess GameRegistry will be our dumping ground :p.

Reply:
Boo!!! More clutter!! Booo!!!!